src/Kerbalism/Utility/DumpSpec.cs

```diff
@@ -69,7 +69,7 @@ public int ValveIndex
 			set
 			{
 				valve_i = (!AnyValves || value > valves.Count - 1 || value < 0) ? 0 : value;
-				DeployValve();
+				if (AnyValves) DeployValve();
```

Comment:
DeployValve() already check if has AnyValues, DeployValve() cleans the valve lists if has no valve.
@PiezPiedPy, you create this Valve system, could you review those changes?

Reply:
It kills also the list of non-valve resources, that was the problem

Reply:
I'll check when I get back from work ;)

Reply:
@madman2003 The reason the list of non-valve resources is killed is because either a `dump` or `dump_valves` parameter is used and not both together. If this is causing a problem you can see could you explain about it